glmmbernoulli.qmd

```diff
@@ -0,0 +1,523 @@
+The probability function for the Bernoulli distribution is $\mu$ for $y=1$ and $1-\mu$ for $y=0$.
+If we write this in a somewhat contrived way as $\mu^y+(1-\mu)^{1-y}$ for $y\in\{0,1\}$ then the logarithm of the probability function becomes
+$$
+\log\left(\mu^y+(1-\mu)^{1-y}\right) = \log(1-\mu) +
+y\,\log\left(\frac{\mu}{1-\mu}\right) .
+$$ {#eq-BernoulliProb}
+Notice that the logit link function is the multiple of $y$ in the last term.
+
+The characteristic of distributions in the exponential family is that the logarithm of the probability mass function or probability density function, whichever is appropriate, can be expressed as a sum of up to three terms: one that involves $y$ only, one that involves the parameters only and the product of $y$ and a function of the parameters.
+This function is the canonical link for the distribution.
+
+In the case of the Poisson distribution the probability function is $\frac{e^{-\mu}\mu^y}{y!}$ for $y\in\{0,1,2,\dots\}$ so the log probability function is
+$$
+-\log(y!)-\mu+y\log(\mu) .
+$$ {#eq-PoissonProb}
+and the canonical link function is $\log(\mu)$.
+
+### Interpreting coefficient estimates {#sec-GLMMcoefficients}
+
+Returning to the interpretation of the estimated coefficients in model `com05` we apply exactly the same interpretation as for a linear mixed model but taking into account that slopes or differences in levels are with respect to the logit or log-odds function.
+If we wish to express results in the probability scale then we should apply the logistic function to whatever combination of coefficients is of interest to us.
+
+```{julia}
+logistic(η) = inv(one(η) + exp(-η))
+newdata = Table(newdata; μ=logistic.(newdata.η))
+```
+
+producing the population predictions on the probability scale, as shown in @fig-com05predmu.
+
+```{julia}
+#| code-fold: true
+#| fig-cap: Predicted probability of contraception use versus centered age from model com05.
+#| label: fig-com05predmu
+draw(
+  data(newdata) *
+  mapping(
+    :age => "Centered age (yr)",
+    :μ => "Probability of contraceptive use";
+    col=:urban => renamer(["N" => "Rural", "Y" => "Urban"]),
+    color=:ch => "Children",
+  ) *
+  smooth();
+  figure=(; resolution=(800, 450)),
+)
+```
+
+On the probability scale we can compare the predictions to the observed frequencies shown with scatterplot smoother lines in @fig-contradata2.
+
+Consider the predictions on both the linear predictor and probability (or expected value) scales for women with centered ages of 2.0.
+
+```{julia}
+filter(r -> r.age == 2, newdata)
+```
+
+The predicted probability of woman with centered age of 2, with children, living in an urban environment using artificial contraception is about 2/3, which is reasonably close to the smoothed frequency for that combination of covariates in @fig-contradata2.
```

Comment:
Predictions haven't been plotted with standard errors so far, but it might be good to mention that you can't just take the inverse link of the standard error to get the error on the response scale for nonlinear links.